owncloudApp/src/main/java/com/owncloud/android/presentation/files/operations/FileOperationsViewModel.kt

```diff
@@ -63,7 +66,8 @@ class FileOperationsViewModel(
     private val setFilesAsAvailableOfflineUseCase: SetFilesAsAvailableOfflineUseCase,
     private val unsetFilesAsAvailableOfflineUseCase: UnsetFilesAsAvailableOfflineUseCase,
     private val contextProvider: ContextProvider,
-    private val coroutinesDispatcherProvider: CoroutinesDispatcherProvider
+    private val coroutinesDispatcherProvider: CoroutinesDispatcherProvider,
+    private val manageDeepLinkUseCase: ManageDeepLinkUseCase,
```

Comment:
Add the use cases as parameters before the providers in the ViewModels:

```suggestion
    private val manageDeepLinkUseCase: ManageDeepLinkUseCase,
    private val contextProvider: ContextProvider,
    private val coroutinesDispatcherProvider: CoroutinesDispatcherProvider,
```

Reply:
Done